apps/web/src/app/[locale]/challenge/[slug]/submissions/[[...catchAll]]/_components/overview.tsx

```diff
@@ -19,13 +20,18 @@ ${code}`;
 
 export function SubmissionOverview({ submissionId }: Props) {
   const { slug } = useParams();
+  const searchParams = useSearchParams();
+
+  const showSuggestions = searchParams.get('success') === 'true';
   const { data: submission } = useQuery({
-    queryKey: ['submission', submissionId],
+    queryKey: [`submission`, submissionId],
```

Comment:
why change to backtick

Reply:
Was trying something maybe forgot to change to single quotes. lol